met-web/src/components/engagement/admin/create/authoring/AuthoringContext.tsx

```diff
@@ -14,12 +14,13 @@ export interface EngagementUpdateData {
     end_date: Dayjs;
     description: string;
     rich_description: string;
+    description_title: string;
     banner_filename: string;
     status_block: string[];
     title: string;
     icon_name: string;
     metadata_value: string;
-    send_report: boolean;
+    send_report: boolean | undefined;
```

Comment:
I need a state for the boolean that is neither true or false, so that if that field isn't set, it isn't submitted in the action. I'm open to suggestions if there's a better way to do it.

Reply:
The question mark means that it's optional; i.e. it can be undefined